.github/workflows/docker-build.yml

```diff
@@ -17,9 +17,29 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v2
 
-    - id: get-repo
-      name: Get Docker Image Identifiers
-      run: echo "::set-output name=IMAGE_REPOSITORY::$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]')"
+    - name: Prepare
+      id: prep
+      run: |
+        DOCKER_IMAGE=${{ env.REGISTRY }}/${{ github.actor }}/${GITHUB_REPOSITORY#*/}
+        VERSION=latest
+        SHORTREF=${GITHUB_SHA::8}
+
+        # If this is git tag, use the tag name as a docker tag
+        if [[ $GITHUB_REF == refs/tags/* ]]; then
+          VERSION=${GITHUB_REF#refs/tags/v}
+        fi
+        TAGS="${DOCKER_IMAGE}:${VERSION},${DOCKER_IMAGE}:${SHORTREF}"
+
+        # If the VERSION looks like a version number, assume that
+        # this is the most recent version of the image and also
+        # tag it 'latest'.
+        if [[ $VERSION =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]]; then
+          TAGS="$TAGS,${DOCKER_IMAGE}:latest"
```

Comment:
Tagging latest here might be bad, as it will keep all old images with the latest tag?

Reply:
Only one image can have specific tag, so if newer image is pushed with tag like latest (or any other tag for that matter) previous image will be untagged. This means that for someone who keeps ddb-importer module up-to-date using latest will always pull newest image, and old images will be kept with version tags if someone stays on older version of the module.